x/dispensation/keeper/grpc_querier.go

```diff
@@ -33,8 +34,8 @@ func (q Querier) ClaimsByType(ctx context.Context,
 
 	claims := q.keeper.GetClaimsByType(sdk.UnwrapSDKContext(ctx), request.UserClaimType)
 	Claims := make([]*types.UserClaim, len(claims))
-	for _, c := range claims {
-		Claims = append(Claims, &c)
+	for index := range claims {
```

Comment:
Did you mean to include this code in the PR?  Looks like it's a dispensation change.

Reply:
It is dispensation code, which block the build, fix it to make the compilation work.